lib/logthrdestdrv.c

```diff
@@ -76,24 +76,21 @@ log_threaded_dest_worker_rewind_messages(LogThreadedDestWorker *self, gint batch
   self->batch_size -= batch_size;
 }
 
-static const gchar *
-_format_queue_persist_name(LogThreadedDestWorker *self)
+static void
+_format_queue_persist_name(LogThreadedDestWorker *self, gchar *persist_name, size_t len)
 {
-  static gchar persist_name[1024];
   LogPipe *owner = &self->owner->super.super.super;
```

Comment:
I would probably convert this (and other similar call sites) to use g_strdup_printf() and ensure the result is freed by the caller.

That way we don't have to care about 1024 character limits.